### DISCUSSION TITLE
Ordre des additionnels

### DISCUSSION THREAD

**Discussion by Bgtb** [witsa/synapps#794] [Q&A] [answered]
Bonjour,
Serait-il possible de réorganiser l'ordre des additionnels ?
En rajoutant des additionnelles, elles se mettent à la suite par rapport à leur création.
Mais pour une utilisation simplifiée, j'aurais besoin de les réorganiser.
Merci

**Comment by WIToliv** [comment, marked as answer]
Bonjour, 
Si vous cliquez sur les engrenages des additionnelles 
![image](https://github.com/user-attachments/assets/5c7ceaae-6538-42a9-bfc1-5ed5ce047c1d)

Vous avez ensuite la possibilité de les montées ou descendre avec les flèches.
![image](https://github.com/user-attachments/assets/53b07beb-5d39-4e91-98b5-c55f178ff6d2)

Espérant que c'était bien votre question :)

**Comment by Bgtb** [comment]
Bonjour,

C’est super !!

Merci beaucoup

Cordialement

Raphael BONIFACIO
Gérant
21 impasse Peyre
64170 SERRES-STE-MARIE
Tél : 0680610266
Mail : ***@***.******@***.***> devient ***@***.***
***@***.***

De : WIToliv ***@***.***>
Envoyé : lundi 11 novembre 2024 16:49
À : witsa/synapps ***@***.***>
Cc : Raphael BONIFACIO ***@***.***>; Author ***@***.***>
Objet : Re: [witsa/synapps] Ordre des additionnels (Discussion #794)


Bonjour,
Si vous cliquez sur les engrenages des additionnelles
image.png (view on web)<https://github.com/user-attachments/assets/5c7ceaae-6538-42a9-bfc1-5ed5ce047c1d>

Vous avez ensuite la possibilité de les montées ou descendre avec les flèches.
image.png (view on web)<https://github.com/user-attachments/assets/53b07beb-5d39-4e91-98b5-c55f178ff6d2>

Espérant que c'était bien votre question :)
